common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -153,6 +153,14 @@ div.problem {
     }
   }
 
+  span > label {
```

Comment:
It looks like this commit should be merged with commit https://github.com/edx/edx-platform/pull/13052/commits/c4fbcddbbc88f789ad7bc32c1b997e94bbc80c15 to get rid of these changes (they all seem to be reverted in that commit).